cli/src/pcluster/aws/common.py

```diff
@@ -184,6 +184,12 @@ def clear_all():
         for cache in Cache._caches:
             cache.clear()
 
+    @staticmethod
+    def add_cache(custom_cache):
+        """Add custom cache, so the cache created outside this wrapper can be cleared to."""
+        # Todo: Validate this approach
```

Comment:
todo?

Reply:
Not valid anymore

I also validate our new approach today